web-console/4.0-designs/overview/overview.md

```diff
@@ -0,0 +1,87 @@
+# Overview
+
+## Collapsed list view
+
+### General
+
+![list view](img/listview.png)
+* The project overview may be viewed in list view or topology view. It will default to show the list view. Users can toggle between the two views by using the view selector icons at the top right corner of the page.
+* Users may filter resources by name by using the filter bar in the top right.
+* The collapsed list view of the project overview will group items by app label (for example, `app=foo`) by default. Users may group by a different label by using the `Group by label` dropdown selector. The selector will be populated with existing labels.
+  * If no resources have labels, items will be shown in a regular list view listed alphabetically by resource type and then name and the `Group by label` dropdown will be hidden.
+* On hover, the row will be highlighted (#EDF8FF) and the icon fa-angle-right (pf-blue-400, #0088CE) will appear at the end of the row. When a row is selected (#DEF3FF), the side panel will open with a summary of that resource's overview, pushing the content over to the left.
+* If a user is not in the context of a selected project, the user will be shown an [empty state](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/empty-states/empty-states) until they select a project from the project dropdown.
+![empty state](img/ES_overview.png)
+
+### List columns
+
+* The first column will include resource badges and names. Users can redirect to a resource's details page by clicking the resource name.
+* The second column will display status updates and other relevant notifications and events such as crash looping containers and deployment failures. Users can hover on a status or notification to see more information.
+* The third column will be a summary of the item's metrics, including memory, CPU, and network. If metrics are not enabled, this column will be blank. If metrics are enabled but are not currently available, users will see "--" in place of the values.
+* The fourth column will show how many pods have deployed for that item. Users can click on the pods link to be directed to the resource's pods page.
+
+### Route URL
+
+![route](img/route.png)
```

Comment:
Can we consider using a normal dialog here to be consistent with other actions?

We added a dialog like this originally for project create in the previous console, and we ended up changing it back to a regular modal.

Reply:
@spadgett That's a good point, I can't think of anywhere else we use a popover. Is the screenshot below more what you were picturing?

![route2](https://user-images.githubusercontent.com/40238481/47165872-f45fa080-d2c8-11e8-9152-51e5052b211d.png)

Reply:
Good point @spadgett makes sense :+1:

Reply:
@bmignano Yeah, that's what I was thinking if you all agree :)

Reply:
if we go with this, we want initial caps for title @bmignano